make/e2e-setup.mk

```diff
@@ -392,17 +393,21 @@ e2e-setup-kyverno: $(call image-tar,kyverno) $(call image-tar,kyvernopre) load-$
 		--wait \
 		--namespace kyverno \
 		--create-namespace \
-		--version 3.0.4 \
-		--set image.tag=v1.8.1 \
-		--set initImage.tag=v1.8.1 \
-		--set image.pullPolicy=Never \
-		--set initImage.pullPolicy=Never \
+		--version 3.2.4 \
+		--set webhooksCleanup.enabled=false \
+		--set reportsController.enabled=false \
+		--set cleanupController.enabled=false \
+		--set backgroundController.enabled=false \
+		--set admissionController.container.image.tag=$(TAG) \
+		--set admissionController.container.image.pullPolicy=Never \
+		--set admissionController.initContainer.image.tag=$(TAG) \
+		--set admissionController.initContainer.image.pullPolicy=Never \
 		kyverno kyverno/kyverno >/dev/null
 	@$(KUBECTL) create ns cert-manager >/dev/null 2>&1 || true
 	$(KUBECTL) apply --server-side -f make/config/kyverno/policy.yaml >/dev/null
 
 $(bin_dir)/downloaded/pebble-$(PEBBLE_COMMIT).tar.gz: | $(bin_dir)/downloaded
-	$(CURL) https://github.com/letsencrypt/pebble/archive/$(PEBBLE_COMMIT).tar.gz -o $@
+	$(CURL) https://github.com/inteon/pebble/archive/$(PEBBLE_COMMIT).tar.gz -o $@
```

Comment:
Why?

Reply:
So, I guess we should downgrade again or fix this in another manner.